src/wasm/extract.cpp

```diff
@@ -126,7 +132,7 @@ bool file_output::calculate_checksum() {
 
 crypto::checksum file_output::checksum() { return checksum_.finalize(); }
 
-Context::Context() : zip_("innoout.zip") { color::init(color::disable, color::disable); }
+Context::Context() { color::init(color::disable, color::disable); }
```

Comment:
Some members are missing an initializer. Consider providing them either here or in the header.